spec/system/user_activity_bookmarks_spec.rb

```diff
@@ -58,4 +58,10 @@
     expect(user_activity_bookmarks).to have_topic(bookmark_1.bookmarkable.topic)
     expect(user_activity_bookmarks).to have_topic(bookmark_2.bookmarkable.topic)
   end
+
+  it "can clear the query with backspace" do
+    user_activity_bookmarks.visit(current_user, q: "dog")
+    user_activity_bookmarks.clear_query_with_backspace
+    expect(user_activity_bookmarks.has_empty_search?).to eq(true)
```

Comment:
```suggestion
    expect(user_activity_bookmarks).to have_empty_search
```

Slight improve in readability which does the same thing.

Reply:
Right, better! Looks like you were faster in updating it!